src/services/API/User.ts

```diff
@@ -0,0 +1,17 @@
+import { apiClient } from "./RootClient";
+import { externalAuthTokenEndpoint } from "./Endpoints";
+import {
+  UserAPIResponse
```

Comment:
It shouldn't be called `UserAPIResponse`. Like I said above, the quiz has only info about whether an org key is valid or not. So this API has to do only with the org